applications/GeoMechanicsApplication/custom_elements/geo_structural_base_element.cpp

```diff
@@ -268,22 +269,18 @@ void GeoStructuralBaseElement<TDim, TNumNodes>::CalculateDampingMatrix(MatrixTyp
     // Rayleigh Method: Damping Matrix = alpha*M + beta*K
 
     // Compute Mass Matrix
-    MatrixType MassMatrix(N_DOF_ELEMENT, N_DOF_ELEMENT);
-
-    this->CalculateMassMatrix(MassMatrix, rCurrentProcessInfo);
+    MatrixType mass_matrix(N_DOF_ELEMENT, N_DOF_ELEMENT);
+    this->CalculateMassMatrix(mass_matrix, rCurrentProcessInfo);
 
     // Compute Stiffness matrix
-    MatrixType StiffnessMatrix(N_DOF_ELEMENT, N_DOF_ELEMENT);
-
-    this->CalculateStiffnessMatrix(StiffnessMatrix, rCurrentProcessInfo);
+    MatrixType stiffness_matrix(N_DOF_ELEMENT, N_DOF_ELEMENT);
+    this->CalculateStiffnessMatrix(stiffness_matrix, rCurrentProcessInfo);
 
     // Compute Damping Matrix
     if (rDampingMatrix.size1() != N_DOF_ELEMENT)
         rDampingMatrix.resize(N_DOF_ELEMENT, N_DOF_ELEMENT, false);
-    noalias(rDampingMatrix) = ZeroMatrix(N_DOF_ELEMENT, N_DOF_ELEMENT);
-
-    noalias(rDampingMatrix) += rCurrentProcessInfo[RAYLEIGH_ALPHA] * MassMatrix;
-    noalias(rDampingMatrix) += rCurrentProcessInfo[RAYLEIGH_BETA] * StiffnessMatrix;
+    noalias(rDampingMatrix) = GeoEquationOfMotionUtilities::CalculateDampingMatrix(
+        rCurrentProcessInfo[RAYLEIGH_ALPHA], rCurrentProcessInfo[RAYLEIGH_BETA], mass_matrix, stiffness_matrix);
```

Comment:
Apparently, the damping for the "structural" elements within geo have an implementation that differs from the continuum elements. I wonder whether we should make the possibility for a material based alpha and beta possible or to avoid that effort as we hope to let the "structural" elements within geo slowly disappear.